net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,197 @@
+id: net.pdl2ork.PdL2Ork
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
+command: pd-wrapper.sh
+finish-args:
+# Display
+  - --share=ipc
+  - --socket=x11
+  #- --socket=wayland
+  - --device=dri
+# Audio
+  - --socket=pulseaudio
+  # currently disabled per a flathub maintainer's request
+  #- --system-talk-name=org.freedesktop.RealtimeKit1
+  - --filesystem=xdg-run/pipewire-0
+# Filesystem access (pd doesn't yet work with portals)
+  - --filesystem=home
```

Comment:
Does this need to full home or you can restrict it to some specific folders like ~/Documents, Downloads etc.?

Reply:
Are you sure the it doesn't support portals? It seems to work with protals from what I can see?

Reply:
I am not sure since this is my first time building a flatpak. I do know that that comment was copied from a manifest of a close fork, so I would tend to trust them over anything I can say. The patches can pull audio files, 3D objects, video files, text files, and bunch of other stuff, and often go beyond that, depending on the needs and imagination of its users, so if I understand the role of portals correctly, I would err on the side of full access. The users can also create folders and save their code/patches, as needed.

Reply:
By default, access is to ~/.pd-l2ork/ ~/Pd-L2Ork (with its subfolders), and ~/pd-l2ork-externals. Users can also save their stuff wherever they like (and where they have permission to do so).

Reply:
No it seems to support portals, my guess would be it needs some kind of persistent access or it is just wrong.

It'd be better if it can be avoided or just restricted to something like 

```
--persist=.pd-l2ork
--filesystem=Pd-L2Ork
--filesystem=pd-l2ork-externals
```

Try running it with `flatpak run --nofilesystem=home io.github.pd_l2ork.Pd_L2Ork` and check the new and open/save dialogues.

Reply:
I think leaving it to `home` is good enough.

Reply:
> Try running it with flatpak run --nofilesystem=home io.github.pd_l2ork.Pd_L2Ork and check the new and open/save dialogues.

@pd-l2ork Did you try this?